tests/test_basic.py

```diff
@@ -1435,8 +1435,8 @@ def test_spikecount_stimint1():
         spikecount_stimint + 2)
 
 
-def test_spikecount_libv4peakindices():
-    """basic: Test Spikecount in combination with LibV4 peak_indices"""
+def test_spikecount_libv5peakindices():
```

Comment:
Shouldn't we just remove this test altogether with the DependencyV5_LibV4peakindices.txt file?
It is testing features that are no longer present.

Reply:
Ok I deleted that and added another test for `setDependencyFileLocation` since we need a test for checking the case when `setDependencyFileLocation` works.